variables.tf

```diff
@@ -154,4 +154,10 @@ variable "vpc_id" {
   description = "Id of the VPC instance where this cluster will be provisioned"
 }
 
-#############################################################################
+variable "confirm_network_healthy" {
```

Comment:
perhaps go with the variable name `verify_worker_network_readiness`

Reply:
Noted. will make the change.